unshare: add page (#7920)
tldr-pages/tldr: pages/linux/unshare.md

# unshare

> > Execute a command in a new namespace.

Reviewer: Since there is more than 1 namespace, e.g the `net`, `mount`, `UTS`, etc etc.
How about:
Thoughts ?
```suggestion
> Execute a command in a new user defined namespace(s).
```

> More information: <https://www.kernel.org/doc/html/latest/userspace-api/unshare.html>.

> - Execute a command without sharing access to connected networks:

Reviewer 2: ```suggestion
- Execute the specified command without sharing access to connected networks:
```

Author: I'm actually partial to this one as well, anyone else care to drop an opinion?

Reviewer 3: a works better.

> `unshare --net {{command}}`

Reviewer 4: I think this clarification is better
```suggestion
`unshare --net {{command}} {{command_arguments}}`
```

Author: I have mixed feelings towards this and favor what we had before.

If anyone else cares to chime in, maybe they can break the tie?

Reviewer 3: I agree with axel.

> - Execute a command as a child process without sharing mounts, processes, or networks:

Reviewer 2: ```suggestion
- Execute the specified command as a child process without sharing mounts, processes, or networks:
```

> `unshare --mount --pid --net --fork {{command}}`

Reviewer 4: ```suggestion
`unshare --mount --pid --net --fork {{command}} {{command_arguments}}`
```
